.github/CODEOWNERS

```diff
@@ -0,0 +1,181 @@
+clang/lib/Analysis/ @llvm/pr-subscribers-clang-analysis
+clang/lib/Basic/Module.cpp @llvm/pr-subscribers-modules
+clang/lib/Basic/Targets/M68k.* @llvm/pr-subscribers-m68k
+clang/lib/CodeGen/** @llvm/pr-subscribers-clang-codegen
+clang/lib/CodeGen/CGCoroutine.cpp @llvm/pr-subscribers-coroutines
+clang/lib/CodeGen/CGDebugInfo.cpp @llvm/pr-subscribers-debug-info
+clang/lib/CodeGen/Targets/M68k.cpp @llvm/pr-subscribers-m68k
+clang/lib/Frontend/ModuleDependencyCollector.cpp @llvm/pr-subscribers-modules
+clang/lib/Frontend/PrecompiledPreamble.cpp @llvm/pr-subscribers-modules
+clang/lib/Lex/ModuleMap.cpp @llvm/pr-subscribers-modules
+clang/lib/Sema/SemaCoroutine.cpp @llvm/pr-subscribers-coroutines
+clang/lib/Sema/SemaModule.cpp @llvm/pr-subscribers-modules
+clang/lib/Serialization/* @llvm/pr-subscribers-modules
+clang/lib/StaticAnalyzer/ @llvm/pr-subscribers-clang-static-analyzer
+clang/test/CXX/module/* @llvm/pr-subscribers-modules
+clang/test/CodeGenCoroutines/* @llvm/pr-subscribers-coroutines
+clang/test/Modules/* @llvm/pr-subscribers-modules
+clang/tools/scan-build/ @llvm/pr-subscribers-clang-static-analyzer
+clang/unittests/Serialization/* @llvm/pr-subscribers-modules
+clang/utils/analyzer/ @llvm/pr-subscribers-clang-static-analyzer
+compiler-rt/lib/memprof/* @llvm/pr-subscribers-pgo
+compiler-rt/lib/profile/* @llvm/pr-subscribers-pgo
+compiler-rt/test/memprof/* @llvm/pr-subscribers-pgo
+compiler-rt/test/profile/* @llvm/pr-subscribers-pgo
+flang/include/flang/Frontend @llvm/pr-subscribers-flang-driver
+flang/include/flang/FrontendTool @llvm/pr-subscribers-flang-driver
+flang/lib/Frontend @llvm/pr-subscribers-flang-driver
+flang/lib/FrontendTool @llvm/pr-subscribers-flang-driver
+flang/test/Driver @llvm/pr-subscribers-flang-driver
+flang/tools/flang-driver @llvm/pr-subscribers-flang-driver
+flang/unittests/Frontend @llvm/pr-subscribers-flang-driver
+libcxx/** @llvm/pr-subscribers-libcxx
+libcxxabi/** @llvm/pr-subscribers-libcxxabi
+libunwind/** @llvm/pr-subscribers-libunwind
+llvm/docs/CommandGuide/FileCheck.rst @llvm/pr-subscribers-testing-tools
+llvm/docs/CommandGuide/lit.rst @llvm/pr-subscribers-testing-tools
+llvm/docs/Coroutines.rst @llvm/pr-subscribers-coroutines
+llvm/docs/TestingGuide.rst @llvm/pr-subscribers-testing-tools
+llvm/include/llvm/BinaryFormat/Dwarf.* @llvm/pr-subscribers-debug-info
+llvm/include/llvm/DebugInfo @llvm/pr-subscribers-debug-info
+llvm/include/llvm/FileCheck/* @llvm/pr-subscribers-testing-tools
+llvm/include/llvm/IR/Debug*.h @llvm/pr-subscribers-debug-info
+llvm/include/llvm/MCA/** @llvm/pr-subscribers-llvm-mca
+llvm/include/llvm/Transforms/Coroutines/* @llvm/pr-subscribers-coroutines
+llvm/lib/CodeGen/AsmPrinter @llvm/pr-subscribers-debug-info
+llvm/lib/DebugInfo @llvm/pr-subscribers-debug-info
+llvm/lib/FileCheck/* @llvm/pr-subscribers-testing-tools
+llvm/lib/IR/Debug*.cpp @llvm/pr-subscribers-debug-info
+llvm/lib/MCA/** @llvm/pr-subscribers-llvm-mca
+llvm/lib/Target/M68k @llvm/pr-subscribers-m68k
```

Comment:
While others backend maintainers haven't requested, you can create all backend teams for all
```
/llvm/lib/Target/$backend/
/llvm/test/CodeGen/$backend/
/clang/lib/Basic/Targets/$backend*
/clang/lib/CodeGen/Targets/$backend
```
as a starter.

Reply:
I noticed on the Discourse post that subscribers for backend teams wouldn't be created automatically.

On behalf of Arm we'd like to follow MaskRays suggestion and add two subscription groups 
$backend = ARM , pr-subscribers-arm
$backend = AArch64, pr-subscribers-aarch64

Will add this to the Discourse post to.